test/functional/admin/case_studies_controller_test.rb

```diff
@@ -19,4 +19,12 @@ class Admin::CaseStudiesControllerTest < ActionController::TestCase
   should_allow_association_with_worldwide_organisations :case_study
   should_allow_association_between_world_locations_and :case_study
   should_allow_association_with_worldwide_priorities :case_study
+
+  view_test "should not display access limited field" do
+    get :new
+
+    assert_select "form#new_edition" do
+      assert_select "input[name*='edition[access_limited]']", false
```

Comment:
did you intentionally use a substring match (`*=` rather than `=`) on the input name here?